packages/migration/src/migrations/test/v0012_setBackgroundTextMode.js

```diff
@@ -32,15 +32,17 @@ describe('setBackgroundTextMode', () => {
                 _test: 'element1',
                 type: 'text',
                 backgroundColor: '#000',
+                content: '',
               },
               {
                 _test: 'element2',
                 type: 'text',
-                backgroundColor: 'transparent',
```

Comment:
Background color can't be `transparent` at this point, it was migrated to `Pattern` in V6